src/pheval/runners/runner.py

```diff
@@ -70,11 +70,11 @@ def pheval_variant_results_dir(self, directory_path):
 
     def build_output_directory_structure(self):
         """build output directory structure"""
-        self.tool_input_commands_dir.mkdir()
-        self.raw_results_dir.mkdir()
-        self.pheval_gene_results_dir.mkdir()
+        self.tool_input_commands_dir.mkdir(exist_ok=True, parents=True)
```

Comment:
Should the runner be creating parent directories, I thought this was handled by the MakeFile?

Reply:
```suggestion
        self.tool_input_commands_dir.mkdir(exist_ok=True)
```

Reply:
Accepted :D